client/src/Components/SearchPageComponents/Filter/index.jsx

```diff
@@ -1,16 +1,29 @@
-import React from "react";
+import React, { useEffect } from "react";
+import axios from "axios";
+import FilterSide from "../FilterSide";
 import CardResultFilter from "../CardResultFilter";
-
 import "./style.css";
 
 export default function Filter() {
+  const [filter, setFilter] = React.useState([]);
+
```

Comment:
add a state for pages
```suggestion
  const [page, setPage] = useState(0)
```
and then set count as `Math.ceil(page/3)`